jetty-util/src/main/java/org/eclipse/jetty/util/security/Credential.java

```diff
@@ -39,10 +37,12 @@
  */
 public abstract class Credential implements Serializable
 {
+    // NOTE: DO NOT INTRODUCE LOGGING TO THIS CLASS
     private static final long serialVersionUID = -7760551052768181572L;
-    private static final Logger LOG = LoggerFactory.getLogger(Credential.class);
-    private static final List<CredentialProvider> CREDENTIAL_PROVIDERS = TypeUtil.serviceProviderStream(ServiceLoader.load(CredentialProvider.class))
-        .flatMap(p -> Stream.of(p.get()))
+    // Intentionally NOT using TypeUtil.serviceProviderStream
+    // as that introduces a Logger requirement that command line Password cannot use.
+    private static final List<CredentialProvider> CREDENTIAL_PROVIDERS = ServiceLoader.load(CredentialProvider.class).stream()
+        .flatMap((p) -> Stream.of(p.get()))
```

Comment:
```suggestion
        .map(ServiceLoader.Provider::get)
```

No need to wrap with `Stream.of()` and then `flatMap()`.